src/assets/error.svg

```diff
@@ -0,0 +1,97 @@
+<svg width="120" height="119" viewBox="0 0 120 119" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This file was not optimized because when doing so it changed the opacity of the image and couldn't figure out how to fix it.